fluent-syntax/src/parser.js

```diff
@@ -134,9 +134,12 @@ export default class FluentParser {
         throw err;
       }
 
-      const errorIndex = ps.index;
+      let errorIndex = ps.index;
       ps.skipToNextEntryStart(entryStartPos);
-      const nextEntryStart = ps.index;
+      let nextEntryStart = ps.index;
```

Comment:
Nit, this can still be a `const`, right?

Reply:
Ah, yes. I stopped using `const` in most of the code I write these days and only use them for actual constants. I typed `let` out of habit. I'll revert to `const` for consistency with the rest of the file.